[2024-12-11T15:19:28.958-0300] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-12-11T15:19:28.966-0300] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: medallion_architecture.task_spark_submit manual__2024-12-11T18:19:22.934287+00:00 [queued]>
[2024-12-11T15:19:28.971-0300] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: medallion_architecture.task_spark_submit manual__2024-12-11T18:19:22.934287+00:00 [queued]>
[2024-12-11T15:19:28.971-0300] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-12-11T15:19:28.982-0300] {taskinstance.py:2327} INFO - Executing <Task(SparkSubmitOperator): task_spark_submit> on 2024-12-11 18:19:22.934287+00:00
[2024-12-11T15:19:28.986-0300] {standard_task_runner.py:63} INFO - Started process 31973 to run task
[2024-12-11T15:19:28.986-0300] {clientserver.py:543} INFO - Closing down clientserver connection
[2024-12-11T15:19:28.990-0300] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'medallion_architecture', 'task_spark_submit', 'manual__2024-12-11T18:19:22.934287+00:00', '--job-id', '52', '--raw', '--subdir', 'DAGS_FOLDER/dag2.py', '--cfg-path', '/tmp/tmpul1o_v_y']
[2024-12-11T15:19:28.991-0300] {standard_task_runner.py:91} INFO - Job 52: Subtask task_spark_submit
[2024-12-11T15:19:29.015-0300] {task_command.py:426} INFO - Running <TaskInstance: medallion_architecture.task_spark_submit manual__2024-12-11T18:19:22.934287+00:00 [running]> on host beatriz-linux
[2024-12-11T15:19:29.079-0300] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='medallion_architecture' AIRFLOW_CTX_TASK_ID='task_spark_submit' AIRFLOW_CTX_EXECUTION_DATE='2024-12-11T18:19:22.934287+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-11T18:19:22.934287+00:00'
[2024-12-11T15:19:29.081-0300] {taskinstance.py:430} INFO - ::endgroup::
[2024-12-11T15:19:29.089-0300] {base.py:84} INFO - Using connection ID 'spark_default' for task execution.
[2024-12-11T15:19:29.090-0300] {spark_submit.py:473} INFO - Spark-Submit cmd: spark-submit --master yarn --conf spark.hadoop.fs.s3a.access.key=ASIAWAA66KBX3IICKJTU --conf spark.hadoop.fs.s3a.secret.key=****** --conf spark.hadoop.fs.s3a.endpoint=s3.amazonaws.com --jars /home/beatriz/spark-jars/hadoop-aws-3.3.4.jar,/home/beatriz/spark-jars/aws-java-sdk-bundle-1.12.508.jar --name arrow-spark --queue root.default /home/beatriz/Documentos/airflow/dags/tarefas/tarefas.py
[2024-12-11T15:19:30.411-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:30 WARN Utils: Your hostname, beatriz-linux resolves to a loopback address: 127.0.1.1; using 192.168.7.5 instead (on interface wlp0s20f3)
[2024-12-11T15:19:30.413-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:30 WARN Utils: Set SPARK_LOCAL_IP if you need to bind to another address
[2024-12-11T15:19:31.683-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:31 INFO SparkContext: Running Spark version 3.5.3
[2024-12-11T15:19:31.683-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:31 INFO SparkContext: OS info Linux, 6.8.0-49-generic, amd64
[2024-12-11T15:19:31.684-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:31 INFO SparkContext: Java version 11.0.25
[2024-12-11T15:19:31.712-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:31 INFO ResourceUtils: ==============================================================
[2024-12-11T15:19:31.712-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:31 INFO ResourceUtils: No custom resources configured for spark.driver.
[2024-12-11T15:19:31.713-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:31 INFO ResourceUtils: ==============================================================
[2024-12-11T15:19:31.713-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:31 INFO SparkContext: Submitted application: Medallion Architecture
[2024-12-11T15:19:31.736-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:31 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2024-12-11T15:19:31.749-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:31 INFO ResourceProfile: Limiting resource is cpus at 1 tasks per executor
[2024-12-11T15:19:31.752-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:31 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2024-12-11T15:19:31.815-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:31 INFO SecurityManager: Changing view acls to: beatriz
[2024-12-11T15:19:31.815-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:31 INFO SecurityManager: Changing modify acls to: beatriz
[2024-12-11T15:19:31.816-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:31 INFO SecurityManager: Changing view acls groups to:
[2024-12-11T15:19:31.816-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:31 INFO SecurityManager: Changing modify acls groups to:
[2024-12-11T15:19:31.817-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:31 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users with view permissions: beatriz; groups with view permissions: EMPTY; users with modify permissions: beatriz; groups with modify permissions: EMPTY
[2024-12-11T15:19:32.019-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:32 INFO Utils: Successfully started service 'sparkDriver' on port 39837.
[2024-12-11T15:19:32.048-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:32 INFO SparkEnv: Registering MapOutputTracker
[2024-12-11T15:19:32.080-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:32 INFO SparkEnv: Registering BlockManagerMaster
[2024-12-11T15:19:32.098-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:32 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2024-12-11T15:19:32.099-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:32 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2024-12-11T15:19:32.130-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:32 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2024-12-11T15:19:32.150-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:32 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-1bf165a8-230c-4092-bf1c-d49d8fae4d81
[2024-12-11T15:19:32.165-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:32 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2024-12-11T15:19:32.212-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:32 INFO SparkEnv: Registering OutputCommitCoordinator
[2024-12-11T15:19:32.422-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:32 INFO JettyUtils: Start Jetty 0.0.0.0:4040 for SparkUI
[2024-12-11T15:19:32.492-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:32 WARN Utils: Service 'SparkUI' could not bind on port 4040. Attempting port 4041.
[2024-12-11T15:19:32.493-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:32 WARN Utils: Service 'SparkUI' could not bind on port 4041. Attempting port 4042.
[2024-12-11T15:19:32.504-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:32 INFO Utils: Successfully started service 'SparkUI' on port 4042.
[2024-12-11T15:19:32.751-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:32 INFO DefaultNoHARMFailoverProxyProvider: Connecting to ResourceManager at /0.0.0.0:8032
[2024-12-11T15:19:34.018-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:34 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:19:35.020-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:35 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:19:36.022-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:36 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:19:37.022-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:37 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:19:38.023-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:38 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:19:39.024-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:39 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:19:40.025-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:40 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:19:41.026-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:41 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:19:42.027-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:42 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:19:43.028-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:43 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:19:44.042-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:44 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:19:45.043-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:45 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:19:46.044-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:46 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:19:47.045-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:47 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:19:48.047-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:48 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:19:49.048-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:49 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:19:50.049-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:50 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:19:51.050-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:51 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:19:52.051-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:52 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:19:53.052-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:53 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:19:53.054-0300] {spark_submit.py:634} INFO - 24/12/11 15:19:53 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 1 failover attempts. Trying to failover after sleeping for 22049ms.
[2024-12-11T15:20:16.106-0300] {spark_submit.py:634} INFO - 24/12/11 15:20:16 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:20:17.107-0300] {spark_submit.py:634} INFO - 24/12/11 15:20:17 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:20:18.108-0300] {spark_submit.py:634} INFO - 24/12/11 15:20:18 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:20:19.110-0300] {spark_submit.py:634} INFO - 24/12/11 15:20:19 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:20:20.111-0300] {spark_submit.py:634} INFO - 24/12/11 15:20:20 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:20:21.113-0300] {spark_submit.py:634} INFO - 24/12/11 15:20:21 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:20:22.114-0300] {spark_submit.py:634} INFO - 24/12/11 15:20:22 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:20:23.115-0300] {spark_submit.py:634} INFO - 24/12/11 15:20:23 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:20:24.117-0300] {spark_submit.py:634} INFO - 24/12/11 15:20:24 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:20:25.118-0300] {spark_submit.py:634} INFO - 24/12/11 15:20:25 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:20:25.120-0300] {spark_submit.py:634} INFO - 24/12/11 15:20:25 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 2 failover attempts. Trying to failover after sleeping for 42472ms.
[2024-12-11T15:21:08.594-0300] {spark_submit.py:634} INFO - 24/12/11 15:21:08 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:21:09.595-0300] {spark_submit.py:634} INFO - 24/12/11 15:21:09 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:21:10.596-0300] {spark_submit.py:634} INFO - 24/12/11 15:21:10 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:21:11.598-0300] {spark_submit.py:634} INFO - 24/12/11 15:21:11 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:21:12.599-0300] {spark_submit.py:634} INFO - 24/12/11 15:21:12 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:21:13.601-0300] {spark_submit.py:634} INFO - 24/12/11 15:21:13 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:21:14.602-0300] {spark_submit.py:634} INFO - 24/12/11 15:21:14 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:21:15.603-0300] {spark_submit.py:634} INFO - 24/12/11 15:21:15 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:21:16.605-0300] {spark_submit.py:634} INFO - 24/12/11 15:21:16 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:21:17.607-0300] {spark_submit.py:634} INFO - 24/12/11 15:21:17 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:21:17.609-0300] {spark_submit.py:634} INFO - 24/12/11 15:21:17 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 3 failover attempts. Trying to failover after sleeping for 37543ms.
[2024-12-11T15:21:56.155-0300] {spark_submit.py:634} INFO - 24/12/11 15:21:56 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:21:57.156-0300] {spark_submit.py:634} INFO - 24/12/11 15:21:57 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:21:58.157-0300] {spark_submit.py:634} INFO - 24/12/11 15:21:58 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:21:59.158-0300] {spark_submit.py:634} INFO - 24/12/11 15:21:59 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:22:00.159-0300] {spark_submit.py:634} INFO - 24/12/11 15:22:00 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:22:01.160-0300] {spark_submit.py:634} INFO - 24/12/11 15:22:01 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:22:02.161-0300] {spark_submit.py:634} INFO - 24/12/11 15:22:02 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:22:03.163-0300] {spark_submit.py:634} INFO - 24/12/11 15:22:03 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:22:04.164-0300] {spark_submit.py:634} INFO - 24/12/11 15:22:04 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:22:05.166-0300] {spark_submit.py:634} INFO - 24/12/11 15:22:05 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:22:05.168-0300] {spark_submit.py:634} INFO - 24/12/11 15:22:05 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 4 failover attempts. Trying to failover after sleeping for 15925ms.
[2024-12-11T15:22:22.096-0300] {spark_submit.py:634} INFO - 24/12/11 15:22:22 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:22:23.097-0300] {spark_submit.py:634} INFO - 24/12/11 15:22:23 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:22:24.099-0300] {spark_submit.py:634} INFO - 24/12/11 15:22:24 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:22:25.100-0300] {spark_submit.py:634} INFO - 24/12/11 15:22:25 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:22:26.102-0300] {spark_submit.py:634} INFO - 24/12/11 15:22:26 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:22:27.104-0300] {spark_submit.py:634} INFO - 24/12/11 15:22:27 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:22:28.105-0300] {spark_submit.py:634} INFO - 24/12/11 15:22:28 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:22:29.107-0300] {spark_submit.py:634} INFO - 24/12/11 15:22:29 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:22:30.108-0300] {spark_submit.py:634} INFO - 24/12/11 15:22:30 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:22:31.109-0300] {spark_submit.py:634} INFO - 24/12/11 15:22:31 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:22:31.111-0300] {spark_submit.py:634} INFO - 24/12/11 15:22:31 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 5 failover attempts. Trying to failover after sleeping for 35334ms.
[2024-12-11T15:23:07.446-0300] {spark_submit.py:634} INFO - 24/12/11 15:23:07 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:23:08.448-0300] {spark_submit.py:634} INFO - 24/12/11 15:23:08 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:23:09.448-0300] {spark_submit.py:634} INFO - 24/12/11 15:23:09 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:23:10.449-0300] {spark_submit.py:634} INFO - 24/12/11 15:23:10 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:23:11.450-0300] {spark_submit.py:634} INFO - 24/12/11 15:23:11 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:23:12.451-0300] {spark_submit.py:634} INFO - 24/12/11 15:23:12 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:23:13.452-0300] {spark_submit.py:634} INFO - 24/12/11 15:23:13 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:23:14.453-0300] {spark_submit.py:634} INFO - 24/12/11 15:23:14 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:23:15.453-0300] {spark_submit.py:634} INFO - 24/12/11 15:23:15 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:23:16.454-0300] {spark_submit.py:634} INFO - 24/12/11 15:23:16 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-11T15:23:16.455-0300] {spark_submit.py:634} INFO - 24/12/11 15:23:16 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 6 failover attempts. Trying to failover after sleeping for 24821ms.
[2024-12-11T15:23:18.864-0300] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-12-11T15:23:18.872-0300] {process_utils.py:132} INFO - Sending 15 to group 31973. PIDs of all processes in the group: [31974, 32061, 31973]
[2024-12-11T15:23:18.873-0300] {process_utils.py:87} INFO - Sending the signal 15 to group 31973
[2024-12-11T15:23:18.874-0300] {taskinstance.py:2607} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-12-11T15:23:18.875-0300] {spark_submit.py:760} INFO - Sending kill signal to spark-submit
[2024-12-11T15:23:18.875-0300] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-12-11T15:23:18.882-0300] {taskinstance.py:2890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/beatriz/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/beatriz/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/beatriz/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
  File "/home/beatriz/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 176, in execute
    self._hook.submit(self.application)
  File "/home/beatriz/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 548, in submit
    self._process_spark_submit_log(iter(self._submit_sp.stdout))  # type: ignore
  File "/home/beatriz/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 597, in _process_spark_submit_log
    for line in itr:
  File "/home/beatriz/Documentos/airflow/airflow_venv/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 2609, in signal_handler
    raise AirflowTaskTerminated("Task received SIGTERM signal")
airflow.exceptions.AirflowTaskTerminated: Task received SIGTERM signal
[2024-12-11T15:23:18.887-0300] {taskinstance.py:1205} INFO - Marking task as FAILED. dag_id=medallion_architecture, task_id=task_spark_submit, execution_date=20241211T181922, start_date=20241211T181928, end_date=20241211T182318
[2024-12-11T15:23:18.926-0300] {process_utils.py:80} INFO - Process psutil.Process(pid=31974, status='terminated', started='15:19:28') (31974) terminated with exit code None
[2024-12-11T15:23:18.927-0300] {process_utils.py:80} INFO - Process psutil.Process(pid=32061, status='terminated', started='15:19:30') (32061) terminated with exit code None
[2024-12-11T15:23:18.927-0300] {process_utils.py:80} INFO - Process psutil.Process(pid=31973, status='terminated', exitcode=2, started='15:19:28') (31973) terminated with exit code 2
